### PR TITLE
Add message formats

### PR DESCRIPTION
**Pull request by amihaila**
What it sounds like

### CODE REVIEW

File: message_types.h [hunk 1]
```diff
@@ -62,4 +47,83 @@
 #define BOARD_ID_VENT             0x0B
 #define BOARD_ID_VENT_SPARE       0x0C
 
+/*
+ * General message type format (from spreadsheet):
+ * (Version 0.0.3)
```
Comment (akmorrison) [inline message_types.h]: This version number does not match the one in can.h

Please change the one in can.h

File: message_types.h [hunk 2]
```diff
@@ -62,4 +47,83 @@
...
+
+// GENERAL_CMD: COMMAND_TYPE
+enum GEN_CMD {
+    BUS_DOWN_WARNING = 0,
+};
+
+// VALVE_CMD/STATUS STATES
+enum VALVE_STATE {
+    VALVE_OPEN = 0,
+    VALVE_CLOSED,
```
Comment (akmorrison) [inline message_types.h]: Does C guarantee consecutivity of enum values? Also do we know that these are representable in 1 byte?

Comment (amihaila) [reply message_types.h]: https://docs.microsoft.com/en-us/cpp/c-language/c-enumeration-declarations?view=vs-2017

yes. Also, I'm pretty sure the number "3" fits inside 1 byte. I'm also pretty sure that we will not come up with 256 valve/board/sensor states.

File: message_types.h (hunk 1 above)
Comment (amihaila) [reply message_types.h]: done

**Pull request closed without merging**